src/components/NavBar/DropdownMenu/index.tsx

```diff
@@ -1,38 +1,73 @@
 import { useState } from 'react';
 import CustomButton from 'components/CustomButton';
 import profilePic from 'assets/profilepic.jpg';
+import LogInModal from 'components/LogInModal';
 import LoggedIn from './LogedinFront';
 import NavItem from '../NavItem';
 import { DROPDOWN } from './constants';
 import styles from './styles.module.scss';
 
 function DropdownMenu() {
   const [logged, setLogged] = useState(false);
+  const [showModal, setShowModal] = useState(false);
+  const [sign, setSign] = useState(false);
 
   const handleLink = () => {
     console.log('links');
   };
+
+  const openLogin = () => {
+    setShowModal(true);
+    setSign(true);
+  };
   const handleLogin = () => {
     setLogged(true);
   };
+  const handleSignin = () => {
```

Comment:
Sign in is the same as login, change it for `Sign up`

Reply:
done